etc/config.xml

```diff
@@ -74,6 +74,18 @@ xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Store:etc/config.xsd">
                 <payment_action>authorize_capture</payment_action>
             </omise_offsite_truemoney>
 
+            <omise_offsite_conveniencestore>
+                <active>0</active>
+                <title>Convenience Store</title>
+                <model>OmiseConveniencestoreAdapter</model>
```

Comment:
Casing seems off in the model name, but I guess it's consistent with what you've done already. `OmiseConvenienceStoreAdapter` would probably have been better

Reply:
followed existing naming.
`<model>OmiseInternetbankingAdapter</model>`

Reply:
but that is something we could refactor at some point.